Cargo.toml

```diff
@@ -40,6 +40,7 @@ serde_json = "~1.0.8"
 [features]
 mock = ["lru_time_cache/fake_clock", "safe_crypto/mock", "parsec/mock", "parsec/malice-detection"]
```

Comment:
I don't think this is what we had in mind... Let's grab a quick chat.

Reply:
After quick chat, further change will make it into a different PR. Good to go :)